examples/calculator/calc_server.cxx

```diff
@@ -152,9 +152,9 @@ void print_status(const std::string& cmd,
         << "current term: "
             << stuff.raft_instance_->get_term() << std::endl
         << "last snapshot log index: "
-            << stuff.sm_->last_snapshot()->get_last_log_idx() << std::endl
+            << ((stuff.sm_->last_snapshot() != nullptr) ? stuff.sm_->last_snapshot()->get_last_log_idx() : 0) << std::endl
         << "last snapshot log term: "
-            << stuff.sm_->last_snapshot()->get_last_log_term() << std::endl
+            << ((stuff.sm_->last_snapshot() != nullptr) ? stuff.sm_->last_snapshot()->get_last_log_term() : 0) << std::endl
```

Comment:
Please break lines at 90 chars
https://github.com/eBay/NuRaft/blob/71c33400a53aa23055733958c41f2461d3d07b49/.clang-format#L7
Also it can be simplified as follows
```c++
(stuff.sm_->last_snapshot() ? stuff.sm_->last_snapshot()->get_last_log_idx() : 0)
```

Reply:
Okay.